### PR TITLE
feat: optimize cli package and add plugin template

### PR DESCRIPTION
**Pull request by ErKeLost**
1. add js plugin template
2. remove chalk dep

**Description:**

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: packages/cli/templates/js-plugin/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { readFileSync } from 'node:fs';
+import type { JsPlugin } from '@farmfe/core';
+
+export default function farmDtsPlugin(options: any): JsPlugin {
+  return {
+    name: '<FARM-JS-PLUGIN-NPM-NAME>',
+    priority: 1000,
```
Comment (wre232114) [inline packages/cli/templates/js-plugin/src/index.ts:7]: do not set default priority

File: packages/cli/templates/js-plugin/package copy.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (wre232114) [inline packages/cli/templates/js-plugin/package copy.json:1]: remove this file

File: packages/cli/templates/js-plugin/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# @farmfe/js-plugin-dts
+
+## 0.2.1
```
Comment (wre232114) [inline packages/cli/templates/js-plugin/CHANGELOG.md:3]: remove these changelog

**Comment by ErKeLost** [comment]
repair completed

**Pull request merged**